### PR TITLE
[10.0][FIX] Removed total hour attandance from button

### PR DESCRIPTION
**Pull request by YogeshMahera-SerpentCS**
- Task #[932](https://www.quartile.co/web#view_type=form&model=project.task&id=932&active_id=932&menu_id=)

### CODE REVIEW

**Review by yostashiro — APPROVED** [review]
LGTM!

Please bump the version to "10.0.1.3.1", and proceed with the rest of the steps.

**Comment by TimLai125** [comment]
@YogeshMahera-SerpentCS Please bump the version of this module https://github.com/qrtl/nrq-custom/blob/7294761d9eb203c9ff8cd890034ddce463339d6a/hr_attendance_timesheet_adj/__manifest__.py#L9

File: hr_timesheet_adj/__manifest__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "description": """
 * Add ondelete attributes to task and project fields in analytic line.
     """,
-    "version": "10.0.1.0.0",
+    "version": "10.0.1.3.1",
```
Comment (TimLai125) [inline hr_timesheet_adj/__manifest__.py]: @YogeshMahera-SerpentCS Please revert this change

**Review by TimLai125 — CHANGES_REQUESTED** [review]

**Comment by YogeshMahera-SerpentCS** [comment]
@TimLai125 Amended changes

**Review by TimLai125 — APPROVED** [review]

**Pull request merged**